content/cloud-computing/concepts/faas/faas.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'FaaS'
+Description: 'Function as a service (FaaS) is a cloud computing model that offers the ability to develop applications without building and maintaining the typical infrastructure needed for application development and deployment.'
```

Comment:
```suggestion
Description: 'Function as a service is a cloud computing model that allows applications to be made without the infrastructure needed for development and deployment.'
```

The `Description` seemed a bit lengthy; this suggestion brings the character-count to exactly 150 characters.